### ISSUE TITLE
For the function "rotateAndNormalizePoint", what is its function?

### ISSUE THREAD

**Issue by tflucas** [frobelbest/GSLAM#5]
Hello, for the function "rotateAndNormalizePoint", what is its function. I guess it is utilized to process rolling shutter camera.

**Comment by frobelbest** [comment]
Yes. It is to correct rolling shutter effect with given rotation and
normalize the point to unit vectors.

**Comment by tflucas** [comment]
Hello, I have rewritten the code of initialization.
When I test the code in EuRoC dataset, the second largest singular value of the matrix M is not 100x smaller than the largest singular value. 
I can not find where the problem is. Could you give me some advice? Thank you.
